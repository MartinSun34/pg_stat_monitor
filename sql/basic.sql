CREATE EXTENSION pg_stat_monitor;
SELECT pg_stat_monitor_reset();
select pg_sleep(.5);
SELECT 1;
SELECT query FROM pg_stat_monitor ORDER BY query COLLATE "C";
SELECT pg_stat_monitor_reset();
DROP EXTENSION pg_stat_monitor;
