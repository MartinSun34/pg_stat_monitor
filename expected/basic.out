CREATE EXTENSION pg_stat_monitor;
SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

select pg_sleep(.5);
 pg_sleep 
----------
 
(1 row)

SELECT 1;
 ?column? 
----------
        1
(1 row)

SELECT query FROM pg_stat_monitor ORDER BY query COLLATE "C";
             query              
--------------------------------
 SELECT $1
 SELECT pg_stat_monitor_reset()
 select pg_sleep($1)
(3 rows)

SELECT pg_stat_monitor_reset();
 pg_stat_monitor_reset 
-----------------------
 
(1 row)

DROP EXTENSION pg_stat_monitor;
